tests/PHPUnit/System/expected/test_DataComparisonTest_flat__Actions.getPageUrls_month.xml

```diff
@@ -140,10 +140,10 @@
 				<sum_bandwidth_change_from>+800%</sum_bandwidth_change_from>
 				<nb_hits_with_bandwidth_change>-88.9%</nb_hits_with_bandwidth_change>
 				<nb_hits_with_bandwidth_change_from>+800%</nb_hits_with_bandwidth_change_from>
-				<min_bandwidth_change>-75%</min_bandwidth_change>
-				<min_bandwidth_change_from>+300%</min_bandwidth_change_from>
-				<max_bandwidth_change>-75%</max_bandwidth_change>
-				<max_bandwidth_change_from>+300%</max_bandwidth_change_from>
+				<min_bandwidth_change>+0%</min_bandwidth_change>
```

Comment:
Percentage does not work anymore:
`'75%'` is not numeric.

- `php -r "echo (is_numeric('75%') ? 'TRUE' : 'FALSE');"` returns `FALSE`
- `php -r "echo (is_numeric('75foo') ? 'TRUE' : 'FALSE');"` returns `FALSE`

Whereas `min()`/`max()` does the comparison somehow and then returns the original type and value:

- `php -r "echo min('75%', '-20%');"` returns `'-20%'`
- `php -r "echo min('75foo', '-20bar');"` returns `'-20bar'`

See https://www.php.net/manual/en/function.min.php
> Values of different types will be compared using the standard comparison rules. For instance, a non-numeric string will be compared to an int as though it were 0, but multiple non-numeric string values will be compared alphanumerically. **The actual value returned will be of the original type with no conversion applied.**

Reply:
I'll check if using `null === $thisColumnValue || false === $thisColumnValue || '' === $thisColumnValue` instead of `!is_numeric($thisColumnValue)` works in that case

Reply:
Percentage min/max calculation is bugged anyway.
This is caused by ''multiple non-numeric string values will be compared alphanumerically".
Thereby min/max does compare the values of percentage-strings not as we might expect.

- `php -r "echo min('+99%','-20%');"` returns `'+99%'` instead of `'-20%'`
- `php -r "echo min('10%','+20%');"` returns `'+20%'` instead of `'10%'`

Reply:
actually I think that the `change` values should not be aggregated, but should be calculated by a filter. So there might be another issue

Reply:
Looking a bit deeper into the test fixture it seems that there are only visits tracked with a bandwidth of `3574`. So that means that the min and max values is always `3574`. So comparing different datasets indeed has a change of `0%`. So the values was incorrect before, as it was comparing the incorrectly summed up min/max values.

Reply:
If this feature does not calculate with percentage string-values we might use:
`!is_numeric($thisColumnValue)`
instead of
`null === $thisColumnValue || false === $thisColumnValue || '' === $thisColumnValue`

A string that contains some num format is numeric:
- `php -r "echo (is_numeric('-0.5') ? 'TRUE' : 'FALSE');"` returns `TRUE`

A string that contains other characters is not numeric but also does not work correctly with min/max anyway:
- `php -r "echo (is_numeric('10bar') ? 'TRUE' : 'FALSE');"` returns `FALSE`
- `php -r "echo min('+99foo','-20foo');"` returns `'+99foo'` instead of `'-20foo'`
- `php -r "echo min('10bar','+20bar');"` returns `'+20bar'` instead of `'10bar'`